src/main/java/org/truffleruby/options/OptionsBuilder.java

```diff
@@ -48,15 +49,15 @@ public void set(Map<String, Object> properties) {
 
     public void set(OptionValues optionValues) {
         for (OptionDescription<?> option : OptionsCatalog.allDescriptions()) {
-            // TODO CS 19-Jun-17 default value of null isn't supported
-
-            if (option.getDefaultValue() != null) {
-                set(option.getName(), optionValues.get(optionValues.getDescriptors().get(option.getSDKName()).getKey()));
-            }
+            set(option.getName(), optionValues.get(optionValues.getDescriptors().get(option.getSDKName()).getKey()));
```

Comment:
Most of this code will go in the next commit when I properly switch to the new options.